File: app/src/test/java/de/jodamob/kotlin/droidcon/d/RadioState1Test.java
package de.jodamob.kotlin.droidcon.d;

import org.junit.Test;

import java.util.HashMap;
import java.util.Map;

import static junit.framework.Assert.assertTrue;

public class RadioState1Test {

    @Test
    public void should_calculate_strengths() {

        Map<RadioState.SignalTypes, Integer> signalStrength = new HashMap<>();
        signalStrength.put(RadioState.SignalTypes.EDGE, 90);
        signalStrength.put(RadioState.SignalTypes.WIFI, 0);

        assertTrue(RadioState.isSignalStrongEnough(signalStrength));
    }
}

File: app/src/test/java/de/jodamob/kotlin/droidcon/d/RadioState1Test.kt
package de.jodamob.kotlin.droidcon.d

import de.jodamob.kotlin.droidcon.d.RadioState.SignalTypes.EDGE
import de.jodamob.kotlin.droidcon.d.RadioState.SignalTypes.WIFI
import junit.framework.Assert.assertTrue
import org.junit.Test

class RadioState1Test {

    @Test
    fun should_calculate_strengths() {

        val signalStrength = mapOf(EDGE to 90, WIFI to 0)
        assertTrue(RadioState.isSignalStrongEnough(signalStrength))
    }
}